### DISCUSSION TITLE
Links are blue in javadoc comments. Is this a bug?

### DISCUSSION THREAD

**Discussion by PavelTurk** [apache/netbeans#6735] [General]
Links have blue color in javadoc comments in NB 19. Is this a bug it was done purposely?

![Screenshot from 2023-11-21 19-50-42](https://github.com/apache/netbeans/assets/10261361/ad7b27f6-bc02-4d6a-8084-4b703d9e1b5b)

**Comment by mbien** [comment]
yes, the editor renders URIs/URLs in typical link colors. You can activate them with `ctrl+click`. Its theme dependent and can be customized.
![image](https://github.com/apache/netbeans/assets/114367/e2fe0290-5c93-462f-b21b-d303c96c95b1)

**Comment by PavelTurk** [comment]
@mbien Thank you very much.

**Reply by mbien** [reply]
please leave discussions open, you can mark the answer if you want but closing it removes everything

**Comment by PavelTurk** [comment]
@mbien I have a problem with your solution. In order to hide links in Java comments I changed link color to grey. However, now links are grey everywhere (for example in markdown file) as I can't set their colors only for Java. Any ideas?
